src/cryptography/hazmat/primitives/serialization/ssh.py

```diff
@@ -617,6 +617,9 @@ def load_ssh_private_key(
     data: bytes,
     password: typing.Optional[bytes],
     backend: typing.Any = None,
+    comment_collector: typing.Union[
+        typing.Callable[[bytes], None], None
+    ] = None,
```

Comment:
This API is pretty out of step with anything we have elsewhere. Are there any ways we can avoid a callback like this?

Reply:
another way is to save the comments in a runtime attribute of the loaded private key